Sources/Purchasing/SimulatedStore/SimulatedStorePurchaseHandler.swift

```diff
@@ -64,7 +64,6 @@ actor SimulatedStorePurchaseHandler: SimulatedStorePurchaseHandlerType {
             case .error(let error):
                 purchaseResult = .failure(error)
             case .simulateFailure:
-                Logger.debug(Strings.purchase.simulating_purchase_failure)
```

Comment:
I've removed this log as it's redundant with the error log produced by `self.simulatedError`